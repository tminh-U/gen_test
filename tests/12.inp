7
12 20 10 11 15 16 15 
14
3 4
4 5
6 7
1 2
6 6
2 7
7 7
7 7
7 7
4 7
4 6
4 5
1 6
2 5
