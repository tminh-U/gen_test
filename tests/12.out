4
4
3
4
2
25
2
2
2
12
8
4
22
9
